common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -354,12 +361,12 @@ def transcript(self, request, dispatch):
                 response = Response(
                     transcript_content,
                     headerlist=[
-                        ('Content-Disposition', 'attachment; filename="{}"'.format(transcript_filename.encode('utf8'))),
-                        ('Content-Language', self.transcript_language),
+                        ('Content-Disposition', 'attachment; filename="{}"'.format(filename.encode('utf8'))),
+                        ('Content-Language', lang),
                     ],
                     charset='utf8'
                 )
-                response.content_type = transcript_mime_type
+                response.content_type = mime_type
 
         elif dispatch.startswith('available_translations'):
```

Comment:
Changes in this file are different from what we did in https://github.com/edx/edx-platform/pull/17718. Is this intentionally? Changes in the https://github.com/edx/edx-platform/pull/17718 are final and should be used.

Reply:
@muhammad-ammar this is temporary just to make sure that our contentstore part of the interface works as smooth as it should be! :)